src/components/AppComponent.tsx

```diff
@@ -184,7 +185,12 @@ const AppComponent: FC = () => {
   const Modal = showModal ? modals[showModal] : null
 
   return (
-    <div>
+    <div
+      style={{
+        /* spacing.safeAreaTop applies for rounded screens */
+        paddingTop: token('spacing.safeAreaTop'),
+      }}
```

Comment:
Does the `css` function work here? I think tokens can be used directly in `css` styles. 

The `style` property would generally be used only for values that can only be produced at runtime.

```ts
className={css({
  /* spacing.safeAreaTop applies for rounded screens */
  paddingTop: 'safeAreaTop'
}))
```

Reply:
Ah, good catch! There was previously a case where I needed to override a dynamic `style` and then inadvertently continued using that pattern. I have fixed this here: a35715dc3096eb6e70c53403e058f2df600f9312